### PR TITLE
add LIC 6, documentation and test case, fixes #34

### LINKED ISSUE #34: Implement LIC 6 + Tests

### PR DESCRIPTION
**Pull request by Anthial**

### CODE REVIEW

File: src/Test/LicTest.java [hunk 1]
```diff
@@ -63,6 +63,51 @@ void licFive() {
...
+                new Point2D.Double(0, 0)
+        };
+        params.setPoints(points);
+        assertFalse(Lic.licSix(params));
+
+        //First and last point the same, one point further than dist away from the coincident point
+        Point2D.Double[] points2 = {
+                new Point2D.Double(0, 0 ),
+                new Point2D.Double(0,0),
+                new Point2D.Double(0, params.getDist()*2),
```
Comment (valsen) [inline src/Test/LicTest.java]: the points on line 81 and 82 should switch places for this test, so that the outlier point is the one immediately preceding the last point. This asserts that licSix doesn't end the inner for-loop too early
```suggestion
                new Point2D.Double(0, 0),
```

File: src/Test/LicTest.java [hunk 2]
```diff
@@ -63,6 +63,51 @@ void licFive() {
...
+                new Point2D.Double(0,0),
```
Comment (valsen) [inline src/Test/LicTest.java]: puts the outlier point at the end of the inner for-loop of licSix. Also reduced its distance from the coincident point to make it more like an edge case.
```suggestion
                new Point2D.Double(0,params.getDist() + 0.1),
```
